server/server/src/config.rs

```diff
@@ -213,6 +214,36 @@ impl BillingConfig {
             ),
             google: GoogleConfig::from_env_vars(),
             stripe: StripeConfig::from_env_vars(),
+            apple: AppleConfig::from_env_vars(),
+        }
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct AppleConfig {
+    pub iap_key: String,
+    pub iap_key_id: String,
+    pub asc_public_key: String,
+    pub issuer_id: String,
+    pub subscription_product_id: String,
+    pub asc_shared_secret: String,
+    pub apple_root_cert: Vec<u8>,
+}
+
+impl AppleConfig {
+    pub fn from_env_vars() -> Self {
+        let apple_root_cert_dest = env_or_empty("APPLE_ROOT_CERT_PATH");
+
+        Self {
+            iap_key: env_or_panic("APPLE_IAP_KEY"),
+            iap_key_id: env_or_panic("APPLE_IAP_KEY_ID"),
+            asc_public_key: env_or_panic("APPLE_APP_STORE_CONNECT_PUBLIC_KEY"),
+            issuer_id: env_or_panic("APPLE_ISSUER_ID"),
+            subscription_product_id: env_or_panic("APPLE_SUBSCRIPTION_PRODUCT_ID"),
+            asc_shared_secret: env_or_panic("APP_STORE_CONNECT_SHARED_SECRET"),
+            apple_root_cert: apple_root_cert_dest
```

Comment:
how large is this? Can't just stuff it in an env var right?

Reply:
Oh it's non-utf-8 or something right?

Reply:
We could base64 encode it or something, want to avoid random files we have to keep track of.